CONTRIBUTING.md

```diff
@@ -0,0 +1,122 @@
+# How to Contribute to Jaeger
+
+We'd love your help!
+
+Jaeger is [Apache 2.0 licensed](LICENSE) and accepts contributions via GitHub
+pull requests. This document outlines some of the conventions on development
+workflow, commit message formatting, contact points and other resources to make
+it easier to get your contribution accepted.
+
+We gratefully welcome improvements to documentation as well as to code.
+
+# Certificate of Origin
+
+By contributing to this project you agree to the [Developer Certificate of
+Origin](https://developercertificate.org/) (DCO). This document was created
+by the Linux Kernel community and is a simple statement that you, as a
+contributor, have the legal right to make the contribution. See the [DCO](DCO)
+file for details.
+
+## Making A Change
+
+*Before making any significant changes, please [open an
+issue](https://github.com/jaegertracing/documentation/issues).* Discussing your proposed
+changes ahead of time will make the contribution process smooth for everyone.
+
+Once we've discussed your changes and you've made your updates, then open your PR. Your
+pull request is most likely to be accepted if it:
+
+* Has a [good commit
```

Comment:
Had to think about this a bit, because I wanted to keep the list short so it's not overwhelming.  ;-)

- Uses the active voice (vs. passive voice) to make it clear when the user has to perform an action and when actions happen automatically. (could link “passive voice” to my favorite passive voice test https://www.grammarly.com/blog/a-scary-easy-way-to-help-you-find-passive-voice/)
- Has been spellchecked and uses proper grammar.  (Could link the word “grammar” to https://www.grammarly.com/)
- Has been written in plain language and avoids jargon.   Remember, the people reading the documentation know much less about the project than you do.

Reply:
we have been referring to http://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html from all other Jaeger repos. If we want to write our own summary, I would put it in a separate file and refer to it from other repos.

Reply:
@yurishkuro since this is the Documentation repo, these tips were for writing better documentation, not how to write better commit messages.

Reply:
hah! +1

Reply:
@JStickler thanks!